github_changelog_generator.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency("github_api", ["~> 0.12"])
   spec.add_runtime_dependency("colorize", ["~> 0.7"])
+  spec.add_runtime_dependency("sqlite3")
```

Comment:
I could try replacing it with http://ruby-doc.org/stdlib-2.1.3/libdoc/yaml/rdoc/YAML/Store.html . I don't know the performance characteristics of YAML::Store, but it wouldn't need any gems.